command.go

```diff
@@ -264,8 +265,15 @@ func stripFlags(args []string) []string {
 				inQuote = true
 			case strings.Contains(y, "=\""):
 				inQuote = true
+			case strings.HasPrefix(y, "--") && !strings.Contains(y, "="):
```

Comment:
I think this will not work if I use `--boolflag`, in which case the second word should actually be the command, right?